src/ORM/DataList.php

```diff
@@ -1625,12 +1617,6 @@ public function setQueriedColumns($queriedColumns)
         });
     }
 
-    /**
-     * Filter this list to only contain the given Primary IDs
-     *
-     * @param array $ids Array of integers
-     * @return $this
```

Comment:
Why is this docblock being removed?  And shouldn't the return type be `static<T>`?

Reply:
This docblock doesn't differ at all from the superclass or interface it's inheriting the method from. No need to repeat it.
https://github.com/silverstripe/silverstripe-framework/pull/11108#discussion_r1451866912